dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      bundle exec rake db:structure:verify
```

Comment:
> Ana06 11 minutes ago
> That's what we agreed on in our standup.
> We agreed to have this, not that it should be with the style linters. Or at least I din't > understand that. :confused:

I am quite sure that Henne proposed to put it to the linter tests.

This also has the nice advantage that we don't have to handle switching the test coverage on and off. Which I suspect we don't do properly right now (#3294).